bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/product/Product.java

```diff
@@ -34,17 +34,17 @@
 @ConsumerType
 public interface Product extends Component, PageMetadata {
 
-    enum Section {
-        TITLE,
-        PRICE,
-        SKU,
-        IMAGE,
-        OPTIONS,
-        QUANTITY,
-        ACTIONS,
-        DESCRIPTION,
-        DETAILS
-    }
+    String[] sections = new String[] {
```

Comment:
Please define constant for each section as well so that custom implementations can refer to them without relying on the index of this array / hardcoding the constant value.

Also please follow the java naming conventions for constant values:

```
String TITLE_SECTION = "TITLE"
```